templates/drop/config_drop.tmpl

```diff
@@ -1,4 +1,4 @@
-# templates for LRA generation. Please modify everything between {{ }} according to your configuration
+# templates for DROP-generated outputs. Please modify everything between {{ }} according to your configuration
```

Comment:
I wondering if you could duplicate this file, by providing a `config_drop.tmpl` with some more advanced options, and keep the `config_lra.tmpl` for the standard LRA usage.